lib/model/ff_exhibition.dart

```diff
@@ -277,6 +277,20 @@ class Post {
       };
 }
 
+class AdditionalInfo {
```

Comment:
please make class name more clearly. And also `readMore`.

Reply:
I think `AdditionalInfo` is clear enough. Do you suggest another word?